### PR TITLE
helm chart creation for uat and deploy [PAGOPA 351]

### PR DESCRIPTION
**Pull request by AngeloCaporaso**
#### List of Changes

- helm chart adjustment adding startup probe

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by pasqualespica — APPROVED** [review]
great 👍

File: helm/values-uat.yaml [hunk 1]
```diff
@@ -1,25 +1,31 @@
...
     periodSeconds: 10
   readinessProbe:
     httpGet:
-      path: /actuator/health/readiness
+      path: /info
       port: 8080
     initialDelaySeconds: 90
     failureThreshold: 6
     periodSeconds: 10
+  startupProbe:
```
Comment (pasqualespica) [inline helm/values-uat.yaml:23]: i'm not sure this is necessary, see [here](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-readiness-probes)

**Review by pasqualespica — APPROVED** [review]

File: helm/values-uat.yaml (hunk 1 above)
Comment (FedericoRuzzier) [reply helm/values-uat.yaml:23]: Our doubt is that if readiness and liveness execute simultaneously, delay may be missed and an early restart initiated

Comment (AngeloCaporaso) [reply helm/values-uat.yaml:23]: Furthermore, probably, taking into account the failing startup scenario, the effect of startup probe is not comparable with the action triggered by readiness probe. The last one effect is the inbound traffic stop, while start probe, trigger  pod’s restartPolicy.

So our doubt is this: if we consider the scenario without startup probes and with a very long startup phase, readiness interrupts traffic and liveness restart the pod, seems that this behaviour does not solve the problem of the wait necessary for startup.

If the above scenario is yet covered by readiness and liveness probe, startup probe is useless.

**Pull request merged**